setup.py

```diff
@@ -67,36 +67,36 @@
 
 install_requires = [
     # General Invenio dependencies
-    "invenio-app==1.3.0",
-    "invenio-base==1.3.0",
-    "invenio-config==1.0.3",
+    "invenio-app>=1.3.0,<2.0.0",
+    "invenio-base>=1.3.0,<2.0.0",
+    "invenio-config>=1.0.3,<2.0.0",
     # Custom Invenio `base` bundle
-    "invenio-assets==3.0.0",
-    "invenio-accounts==3.0.0",
     "importlib-metadata==4.13.0",
-    "invenio-logging[sentry]==1.3.0",
+    "invenio-accounts>=3.0.0,<4.0.0",
+    "invenio-assets>=3.0.0,<4.0.0",
+    "invenio-logging[sentry]>=2.1.0,<3.0.0",
     "invenio-rest==1.2.8",
```

Comment:
This one was not changed. The latest version requires a new mayor of invenio-logging, and the sentry dependencies are different